src/common/components/help/help-install-snap.js

```diff
@@ -21,11 +21,17 @@ export default class HelpInstallSnap extends Component {
       <div className={styles.helpFlexWrapper}>
         <HeadingThree className={styles.helpFlexHeading}>{ headline }</HeadingThree>
         <div className={styles.helpText}>
-          <pre>
-            <code className={ styles.cli }>
+          <pre className={styles.cli}>
```

Comment:
These styles need adding everywhere where we show some instructions:

For example when you go to build details page release instructions in the bottom are broken:
<img width="754" alt="screen shot 2017-08-17 at 08 54 48" src="https://user-images.githubusercontent.com/83575/29399551-fad9129c-8329-11e7-8e6d-a0aa8d73f8fb.png">